docs/developers.rst

```diff
@@ -103,46 +103,51 @@ Sometimes it is possible to face a timeout issue when trying to push huge files.
 
 #. Just push the files as usual
 
-Keeping Production Data out of Master Branch
+Production Data And Git Submodules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Production versus test data. Any branch beginning with "production" or "application" will contain data in the "production/" subfolder. This data should stay in that branch and not be merged into master. To keep the data out, the easiest practice is to simply keep merges one-way from master into the production branch and not vice versa.
+Production data is located in separate git repositories each scenario in its own repo (see `#3254 <https://github.com/LBNL-UCB-STI/beam/issues/3254>`_ for the reason)
```

Comment:
I think the link to the issue is not what we need in documentation, instead I assume using the following explanation:
Separation of production data and code is needed to: 
1. reducing the git repository size for developers
2. easier addition or changing production data without merging back into develop code changes
3. ability to use any production data with any code branch/commit without creation of yet another git production branch

Reply:
done